File: Exercicio-02.c
#include <stdio.h>

int main()
{
    int n1, n2;
    
    printf("Digite o primeiro número: ");
    scanf("%d", &n1);
    
    printf("Digite o segundo número: ");
    scanf("%d", &n2);
    
    if (n1 > n2){
        printf("O primeiro numero [%d] é maior!\n", n1);
    } else if (n2 > n1){
        printf("O segundo numero [%d] é maior!\n", n2);
    } else{
        printf("Os numeros são iguais!\n");
    }

    return 0;
}

File: Exercicio-02.java
import java.util.Scanner;

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    int n1, n2;

    System.out.println("Digite o primeiro valor: ");
    n1 = sc.nextInt();

    System.out.println("Digite o segundo valor: ");
    n2 = sc.nextInt();
    
    if (n1 > n2){
        System.out.println("O primeiro numero [" + n1 + "] é maior!\n");
    } else if (n2 > n1) {
        System.out.println("O primeiro numero [" + n2 + "] é maior!\n");
    } else {
        System.out.println("Os numeros são iguais!\n");
    }
  }
}
